hw2/server.js

```diff
@@ -7,6 +8,7 @@ app.use(express.json());
 const PORT = process.env.PORT || 3000;
 
 app.use("/users", userRoutes);
+runDb();

```

Comment:
Connection to database should be established before application is published, this need's to be 'await runDb()',

or runDb().then(rest of code)